### PR TITLE
Publish topics using friendly_name and not IP

### LINKED ISSUE #3: Support for audio groups
At the moment audio groups are not supported because they have the same IP as one of the devices.
Do you think its possible to use the friendly_name instead? 
Also the player_state and some other propeties of the single devices are not correct if i stream something to a group (Maybe this is a limitation of pychromecast?).

### PR DESCRIPTION
**Pull request by berryk**
Publishes topics using friendly_name and not IP. 

This also resolves #3 support for audio groups.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,56 +9,56 @@ Provides status information and control capabilities of your Chromecast devices
 * paho-mqtt
 * Zeroconf
 
-You can install the requirements in their correct versions using `pip3 install -r requirements.txt`.
+You can install the requirements in their correct versions using `pfriendly_name3 install -r requirements.txt`.
```
Comment (nohum) [inline README.md]: Seems like a small typo to me ;)

**Review by nohum — CHANGES_REQUESTED** [review]

**Comment by nohum** [comment]
Nice, thanks! I will give it a try today or tomorrow, but just by looking at the code this looks promising 👍

File: README.md (hunk 1 above)
Comment (berryk) [reply README.md]: Yes - well spotted, search and replace error. Sorry!

**Review by berryk — COMMENTED** [review]
Fixed the typo

**Review by nohum — APPROVED** [review]

**Comment by nohum** [comment]
Tested it and works like a charm. Thanks!

**Pull request merged**